lib/sidekiq_adhoc_job.rb

```diff
@@ -0,0 +1,38 @@
+module SidekiqAdhocJob
+
+  InvalidConfigurationError ||= Class.new(RuntimeError)
+
+  def self.configure
+    @_config = Configuration.new
+    yield @_config
+  end
+
+  def self.config
+    @_config
+  end
+
+  def self.init
+    raise InvalidConfigurationError, 'Must configure before init' unless @_config
```

Comment:
Good point. I will re-think about this one.

Reply:
Instead of just checking `worker_path_pattern`, I created a wrapper method `.configured?`, so if there are more options that are required, there's no need to change the check for valid configuration.

Reply:
nice! 👍